smriprep/cli/run.py

```diff
@@ -28,8 +28,7 @@ def get_parser():
     from pathlib import Path
     from argparse import ArgumentParser
     from argparse import RawTextHelpFormatter
-    from templateflow.api import templates
-    from .utils import ParseTemplates, output_space as _output_space
+    from .utils import ParseTemplates, Space
```

Comment:
```suggestion
    from niworkflows.utils.spaces import Space
```

Reply:
I had to fix this one to make docs build :+1: